x/stake/endblock.go

```diff
@@ -33,6 +38,7 @@ func EndBreatheBlock(ctx sdk.Context, k keeper.Keeper) (validatorUpdates []abci.
 
 	if sdk.IsUpgrade(sdk.LaunchBscUpgrade) && k.ScKeeper != nil {
 		sideChainIds, storePrefixes := k.ScKeeper.GetAllSideChainPrefixes(ctx)
+		accumulatedFee := sdk.ZeroDec()
```

Comment:
I know this fee is for Beacon Chain,  so maybe come up with a better name than `accumulatedFee`,  `accumulatedFee` can be too common, or atleast comment for this variable.

Reply:
make sense, changed to a new name.